tests/test_testing.py

```diff
@@ -434,6 +434,22 @@ def test_it_with_settings_passed_implicit_registry(self):
         config = self._callFUT(hook_zca=False, settings=dict(a=1))
         self.assertEqual(config.registry.settings['a'], 1)
 
+    def test_it_with_unpatched_registry(self):
+        from zope.interface.registry import Components
+
+        class DummyRegistry(Components, dict):
+            pass
+
+        dummy_registry = DummyRegistry()
+        config = self._callFUT(
+            registry=dummy_registry, hook_zca=False, settings=dict(a=1)
+        )
+        self.assertEqual(config.registry.settings['a'], 1)
+        dummy = DummyEvent()
+        self.assertEqual(
```

Comment:
The "or self" semantic is that the identical object is returned:
```suggestion
        self.assertIs(
```

Reply:
yeah ok, it'll be the first usage of `assertIs` in the codebase :-)